arithmetic_analysis/jacobi_iteration_method.py

```diff
@@ -115,6 +115,20 @@ def jacobi_iteration_method(
 
     strictly_diagonally_dominant(table)
 
+    """
+    denom - a list of values along the diagonal
+    val - values of the last column of the table array
+
+    masks - boolean mask of all strings without diagonal
+    elements array coefficient_matrix
+
+    ttt - coefficient_matrix array values without diagonal elements
+    ind - column indexes for each row without diagonal elements
+    arr - list obtained by column indexes from the list init_val
+
+    the code below uses vectorized operations based on
+    the previous algorithm on loopss:

```

Comment:
Why comment out the old code? Why not just delete it?

Reply:
I thought that based on the old example with loops, it is more clear how the algorithm is made. If not, can I delete it?

Reply:
Just in case, I am attaching the jacobi_simple_two.py file, in which you can run the algorithms once by specifying the array size n = 256 # array size 'coefficient'. I am attaching data for array sizes of 256 and 500 and the difference only increases 54 and 56 times faster.
```
n = 256 # array size 'coefficient'

time_old 0:00:14.445305
time_new 0:00:00.264163
difference in time 54.68469389300262
True

n = 500 # array size 'coefficient'

time_old 0:01:53.208590
time_new 0:00:01.997707
difference in time 56.66922360714822
```
[jacobi_simple_two.py.tar.gz](https://github.com/TheAlgorithms/Python/files/12329834/jacobi_simple_two.py.tar.gz)